templates/base.html

```diff
@@ -2,10 +2,18 @@
 <html class="no-js" lang="en"> <!--<![endif]-->
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" type="image/png" href="{{ STATIC_URL }}img/favicon.ico" />
+    <link rel="apple-touch-icon" sizes="180x180" href="{{ STATIC_ICONS_URL }}apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="{{ STATIC_ICONS_URL }}favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="{{ STATIC_ICONS_URL }}favicon-16x16.png" sizes="16x16">
```

Comment:
Could we use just `{% static 'favicon-16x16.png' %}` here and everywhere else and remove STATIC_ICONS_URL ?